data/helpers.d/apt

```diff
@@ -186,7 +186,7 @@ ynh_package_install_from_equivs () {
     (cd "$TMPDIR"
     equivs-build ./control 1> /dev/null
     dpkg --force-depends -i "./${pkgname}_${pkgversion}_all.deb" 2>&1)
-    ynh_package_install -f || ynh_die --message="Unable to install dependencies"
+    ynh_package_install -f || { apt-get check 2>&1; ynh_die --message="Unable to install dependencies"; }
```

Comment:
Maybe a small comment here could be useful to explain the usage of {} as a group of commands without starting a sub shell, that has to end with a ;

Otherwise, a wonderful idea !